src/zenml/cli/stack_recipes.py

```diff
@@ -362,6 +362,8 @@ def get_stack_recipes(
                 raise KeyError(
                     f"Stack recipe {stack_recipe_name} does not exist! "
                     f"Available Stack Recipes: {list(stack_recipe_dict)}"
+                    "If you want to deploy a custom stack recipe available "
```

Comment:
Can we add something here about specifying the path of where the custom stack recipe is ?

Reply:
On a side note, the `-p` help message needs impormvents: It now says "Relative path at which you want to install the stack recipe(s)". It should be "Relative path at which local stack recipe(s) should exist"